lib/readline.js

```diff
@@ -921,6 +921,9 @@ exports.Interface = Interface;
 const KEYPRESS_DECODER = Symbol('keypress-decoder');
 const ESCAPE_DECODER = Symbol('escape-decoder');
 
+let timeoutRef;
+const ESCAPE_CODE_TIMEOUT = 100;
```

Comment:
Set it to `0.7` (in the middle) by default and wait until someone complains. Pretty sure no one ever will.